kotori/daq/graphing/grafana/resources/grafana-target-cratedb.json

```diff
@@ -0,0 +1,10 @@
+{
+    "alias": "{{ alias }}",
+    "format": "table",
+    "resultFormat": "time_series",
+    "tags": {{ tags }},
+    "groupByTags": [],
+    "measurement": "{{ measurement }}",
+    "rawQuery": true,
+    "rawSql": "SELECT $__timeGroupAlias(time, $__interval), MEAN(fields['{{ name }}']) AS {{ alias }} FROM {{ table }} WHERE $__timeFilter(time) GROUP BY time ORDER BY time"
```

Comment:
On behalf of a subsequent iteration, we may also want to demonstrate advanced downsampling using the _largest triangle three buckets (LTTB)_ algorithm, as presented by @hlcianfagna, when possible. See [^1].

[^1]: https://community.crate.io/t/advanced-downsampling-with-the-lttb-algorithm/1287